crates/swc_ecma_minifier/src/compress/optimize/util.rs

```diff
@@ -567,3 +600,87 @@ impl Drop for SynthesizedStmts {
         }
     }
 }
+
+#[derive(Default)]
+struct LabelAnalyzer {
+    label: JsWord,
+    /// If top level is a normal block, labelled break must be preserved
+    top_breakable: bool,
+    count: usize,
+    break_layer: usize,
+    continue_layer: usize,
+}
+
+impl LabelAnalyzer {
+    fn visit_mut_loop(&mut self, n: &mut impl VisitMutWith<LabelAnalyzer>) {
+        self.break_layer += 1;
+        self.continue_layer += 1;
+
+        n.visit_mut_children_with(self);
```

Comment:
Similarly, I think we can skip this? Any `break FOO` we find under here can't be removed (same for continue).